core/sql/parser/sqlparser.y

```diff
@@ -16026,13 +16027,30 @@ exe_util_init_hbase : TOK_INITIALIZE TOK_TRAFODION
 
 		 DDLExpr * de = new(PARSERHEAP()) DDLExpr(TRUE, FALSE, TRUE, FALSE,
                                                           FALSE, FALSE,
-							  FALSE, FALSE, FALSE,
+							  FALSE, FALSE, FALSE, FALSE,
```

Comment:
Lately, when I have been adding new options to existing INITIALIZE commands, instead of add a new parameter to the DDLExpr constructor, I have been adding a new bit to "Flags".  Not sure when a new parameter should be added or when a new option should be defined through the "Flags" bitmap. Adding a new flag does not require changing all calls to DDLExpr.